packages/model/addon/-private/record-state.ts

```diff
@@ -233,6 +233,9 @@ export default class RecordState {
     notifications.subscribe(identity, (identifier: StableRecordIdentifier, type: NotificationType, key?: string) => {
       switch (type) {
         case 'state':
+          this.notify('isSaved');
```

Comment:
@runspired what do you think is the best way to test this? Can try to write ones based of an existing one if you have an example